packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -223,6 +224,11 @@ export const BootDataProvider = ({
     setCachedBootData((cachedData) => updateLocalBootData(cachedData, { exp }));
   }, []);
 
+  gqlClient.setHeader(
+    'content-language',
+    (user as Partial<LoggedUser>)?.language || ContentLanguage.English,
+  );
```

Comment:
This sets the language for gql client for all requests. This is earliest I could push it so we load the language as soon as we know it from boot data.

Reply:
This looks good to me!